### ISSUE TITLE
[Feature Request] Include CB Multipart

### ISSUE THREAD

**Issue by EagleDev91** [ChiselsAndBits/Chisels-and-Bits#710]
I currently saw that CB Multipart (1.16 Port of ForgeMultipart) is available since July 2021. It would be great to get the same possibility of 1.12 to place bits inside other compatible Mod Blocks.

**Comment by marchermans** [comment]
Yep is planned for the rebuild, but i need to stabilize the data first

**Comment by EagleDev91** [comment]
Don't get rushed. It's more important to get a stable version/feature instead of getting it earlier ;)
